### PR TITLE
fix: documentation to edit ROOMS_DATA environmen

### PR DESCRIPTION
**Pull request by enieber**
Based in issue #155

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -41,3 +41,4 @@ And then open chrome at: `http://localhost:8080`
    }
  ]
 ``` 
+4) Configure `ROOMS_SOURCE` to `ENVIRONMEN` to get value
```
Comment (lccezinha) [inline SETUP.md]:
```suggestion
4) Configure `ROOMS_SOURCE` to `ENVIRONMENT` to get value 
```

Comment (lccezinha) [reply SETUP.md]: Hey, @enieber thank you for your contribution. Just a small typo fix as a suggestion.

**Comment by lccezinha** [comment]
@juliemar make sense to add this step as you comment in #155?

**Comment by juliemar** [comment]
@lccezinha I believe is not necessary to document the steps for Heroku configuration, because in the issue I tried to explain how to configure Environment Variables in Heroku. By the way, we are docker compatible, and the user can deploy in other container providers.

**Pull request merged**